modules/http/src/test/java/org/mule/module/http/functional/listener/HttpListenerResponseBuilderTestCase.java

```diff
@@ -86,7 +89,10 @@
     public SystemProperty errorHeadersResponseBuilderPath = new SystemProperty("headersResponseBuilderPath","headersResponseBuilderPath");
 
     @Rule
-    public SystemProperty errorHeaderDuplicatesResponseBuilderPath = new SystemProperty("headerDuplicatesResponseBuilderPath","headerDuplicatesResponseBuilderPath");
+    public SystemProperty errorHeaderDuplicatesResponseBuilderPath = new SystemProperty("errorHeaderDuplicatesResponseBuilderPath","errorHeaderDuplicatesResponseBuilderPath");
```

Comment:
This line wasn't part of the fix but had wrong the path name, so it was testing the same as the "headerDuplicatesResponseBuilderPath" test.